new-dti-website/src/app/products/page.tsx

```diff
@@ -1,10 +1,177 @@
+import Image from 'next/image';
 import ImageCarousel from '../../../components/products/imageCarousel';
-import { products } from '../../../components/products/productIcons.json';
+import Connector from '../../../components/products/lines';
+import products from '../../../components/products/products.json';
+import { Button } from '../../../components/ui/button';
+import { cn } from '../../../lib/utils';
+import RedBlob from '../../../components/blob';
+import FloatingImages, { ImageData } from '../../../components/products/Products';
 
-const Page = () => (
-  <div>
-    <ImageCarousel items={products} />
+export default function Page() {
+  const productIcons = [...products.current, ...products.upcoming].map((product) => ({
+    alt: product.alt,
+    path: product.iconPath
+  }));
+
+  return (
+    <div className="overflow-x-hidden">
+      <div className="flex lg:flex-row flex-col relative lg:max-w-[1186.76px] lg:mx-52 lg:my-24 lg:space-x-20 md:space-y-10 md:max-w-[698px] h-fit md:my-[130px] md:mx-[67px] max-w-[314px] mx-auto md:py-0 py-32">
+        <RedBlob
```

Comment:
On some viewport widths ~1100 px, the hero text is cut off. Seems like a combination of the margin and max-width is making this happen.
![image](https://github.com/user-attachments/assets/6c56ef12-5e69-41c2-870c-b816060e9ad1)

Reply:
Ahh I did not notice that thanks! I'll take a look!